app/controllers/api/v0/admin/configs_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V0
+    module Admin
+      class ConfigsController < ApiController
+        before_action :authorize_admin
```

Comment:
Doesn't this rely on having a logged-in user? I'm probably just misunderstanding something, but I expected a call to `authenticate!` somewhere.